go/grpcweb/options.go

```diff
@@ -76,3 +80,22 @@ func WithAllowedRequestHeaders(headers []string) Option {
 		o.allowedRequestHeaders = headers
 	}
 }
+
+// WithWebsockets allows for handling grpc-web requests of websockets - enabling bidirectional requests.
+//
+// The default behaviour is false, i.e. to disallow websockets
+func WithWebsockets(enableWebsockets bool) Option {
```

Comment:
I'd be keen to just turn this on, it's not a backwards compatibility breaking change, and unless my suggestion to add Websockets to DetectTransport is implemented it won't change anything.